pepper-apis/src/main/java/org/broadinstitute/ddp/service/PdfGenerationService.java

```diff
@@ -720,6 +707,128 @@ private void convertSubstitutionToPdf(AnswerSubstitution substitution,
+                                     List<String> errors, Answer answer) throws IOException {
+        if (answer == null) {
+            return;
+        }
+
+        //passed form contains custom composite template pages. just remove those empty pages
+        int pageCount = form.getPdfDocument().getNumberOfPages();
+        IntStream.range(1, pageCount + 1).forEach(i -> form.getPdfDocument().removePage(i));
+
+        CompositeAnswerSubstitution compositeSubstitution = substitution;
+        List<AnswerRow> compositeAnswers = ((CompositeAnswer) answer).getValue();
+        int currentDocumentIndex = 0;
+        for (AnswerRow answerRow : compositeAnswers) {
+            byte[] renderedCompositePdf = renderCompositePdf(answerRow, compositeSubstitution, template, errors);
+            copyPdfToMasterDoc(renderedCompositePdf, currentDocumentIndex, form.getPdfDocument());
+            currentDocumentIndex++;
+        }
+    }
+
+    private void convertChildSubstitutionToPdf(AnswerSubstitution substitution,
+                                               PdfAcroForm form,
+                                               Answer answer,
+                                               List<String> errors) throws IOException {
+
+        String placeholder = substitution.getPlaceholder();
+        PdfFormField field = form.getField(placeholder);
+        if (field == null) {
+            errors.add(String.format("Could not find Child answer PDFFormField field with name: %s", placeholder));
+            return;
+        }
+
+        field.setFont(PdfFontFactory.createFont());
+
+        switch (substitution.getQuestionType()) {
+            case BOOLEAN:
+                substituteBoolean((BooleanAnswerSubstitution) substitution, answer, field);
+                break;
+            case TEXT:
+                substituteText(answer, field);
+                break;
+            case DATE:
+                substituteDate(field, answer);
+                break;
+            case PICKLIST:
+                substitutePicklist(answer, field);
+                break;
+
+            default:
+                errors.add("tried to use an unsupported answer type " + substitution.getQuestionType());
+                return;
+        }
+    }
+
+    private void substituteBoolean(BooleanAnswerSubstitution substitution, Answer answer, PdfFormField field) {
+        BooleanAnswerSubstitution booleanSubstitution = substitution;
+        Boolean boolValue = answer == null ? null : ((BoolAnswer) answer).getValue();
+        if (boolValue != null) {
+            boolean shouldCheck;
+            if (booleanSubstitution.checkIfFalse()) {
+                shouldCheck = !boolValue;
+            } else {
+                shouldCheck = boolValue;
+            }
+            setIsChecked(field, shouldCheck);
+        }
+    }
+
+    private void substituteDate(PdfFormField field, Answer answer) {
```

Comment:
Can you swap the order of these two parameters to be consistent with other similar methods?